snippets/go/send_onchain.go

```diff
@@ -6,31 +6,32 @@ import (
 	"github.com/breez/breez-sdk-go/breez_sdk"
 )
 
-func GetCurrentFees() {
-	// ANCHOR: estimate-current-reverse-swap-total-fees
-	sendAmountSat := uint64(50_000)
-	reverseSwapFeesRequest := breez_sdk.ReverseSwapFeesRequest{
-		SendAmountSat: &sendAmountSat,
-	}
-	if currentFees, err := sdk.FetchReverseSwapFees(reverseSwapFeesRequest); err == nil {
-		log.Printf("Total estimated fees for reverse swap: %v", currentFees.TotalFees)
+func GetCurrentLimits() {
+	// ANCHOR: get-current-reverse-swap-limits
+	if currentLimits, err := sdk.OnchainPaymentLimits(); err == nil {
+		log.Printf("Minimum amount, in sats: %v", currentLimits.MinSat)
+		log.Printf("Maximum amount, in sats: %v", currentLimits.MaxSat)
 	}
-	// ANCHOR_END: estimate-current-reverse-swap-total-fees
+	// ANCHOR_END: get-current-reverse-swap-limits
 }
 
-func ListCurrentFees(currentFees breez_sdk.ReverseSwapPairInfo) {
-	// ANCHOR: get-current-reverse-swap-min-max
-	log.Printf("Minimum amount, in sats: %v", currentFees.Min)
-	log.Printf("Maximum amount, in sats: %v", currentFees.Max)
-	// ANCHOR_END: get-current-reverse-swap-min-max
-}
+func PreparePayOnchain() {
+	// ANCHOR: prepare-pay-onchain
+	sendAmountSat := uint64(50_000)
+	satPerVbyte := uint32(5)
+
+    req := breez_sdk.PrepareOnchainPaymentRequest{
+        AmountSat:               sendAmountSat,
+        AmountType:              breez_sdk.SwapAmountTypeSend,
+        ClaimTxFeerate:          satPerVbyte,
+    }
```

Comment:
CI is complaining about the formatting here

Reply:
Fixed in #141 and merged into `pre-release`.